.docker/app.dockerfile

```diff
@@ -1,5 +1,5 @@
 FROM php:7.2-fpm
 
 RUN apt-get update && apt-get install -y \
-    mysql-client --no-install-recommends \
```

Comment:
I'd like to suggest changing this to `default-mysql-client` because that way if it changes again we don't have to edit this file. I will still accept this review since this also fixes the problem